layouts/partials/footer.html

```diff
@@ -29,7 +29,7 @@
 <ul class="list-inline mb-0">
   {{ range . }}
   <li class="list-inline-item mx-2 h3" data-toggle="tooltip" data-placement="top" title="{{ .name }}" aria-label="{{ .name }}">
-    <a class="text-white" target="_blank" href="{{ .url }}">
+    <a class="text-white" target="_blank" aria-label="icon" href="{{ .url }}">
```

Comment:
`aria-label` should describe the function or content of the element and allow users to differentiate between multiple elements. Using a hardcoded generic string like "icon" doesn't achieve that goal.

In this case the parent element `li` already has the desired values but the attributes are attached to the wrong element. This appears to be due to the tooltip functionality.

If possible, we should move the `data-*`, `aria-label`, and `title` attributes to the `<a>` tag. You'd need to check that the tooltip still works as expected with this change.

If not, we should add `aria-label={{ .name }}` on the `<a>` so the value reflects the link being rendered instead of using a hardcoded value.

Reply:
Thanks, I will put the `title` attribute, tested locally no errors